sys/net/crosslayer/ng_pktbuf/_pktbuf_static.c

```diff
@@ -165,7 +168,7 @@ void *_pktbuf_internal_alloc(size_t size)
 
     while ((node->next != NULL)
            /* and if space between current and next allocation is not big enough */
-           && ((_start_idx(node->next) - _end_idx(node)) < _total_sz(size))) {
+           && ((_start_idx(node->next) - _end_idx(node)) < _al_sz(_total_sz(size)))) {
```

Comment:
`node->next == NULL` means, the current chunk is the last in the packet buffer, so as long as the packet buffer is big enough, exiting the loop here means the new chunk will appended here. A new node/chunk (a node just describes a chunk that comes immediately after it so the terms are somewhat interchangable) is always created when this function is called (this is the purpose of this function), except of cause if there is no space left (which is checked in https://github.com/RIOT-OS/RIOT/pull/3316/files#diff-2901d789a30fdc0411aca7f3e3ec7249R178).